api/database/seeders/DatabaseSeeder.php

```diff
@@ -52,22 +54,86 @@ public function run()
         $this->call(UserSeederLocal::class);
         $this->call(PoolSeeder::class);
 
+
+        // fill digital_careers no salary, with classifications
+        // first 5 classifications in ClassificationSeeder are IT group, therefore take(5) grabs the five IT classifications
         User::factory([
-             'roles' => [ApiEnums::ROLE_APPLICANT]
-        ])
-            ->count(60)
+            'roles' => [ApiEnums::ROLE_APPLICANT]
+       ])
+            ->count(20)
             ->afterCreating(function (User $user) {
                 $assets = CmoAsset::inRandomOrder()->limit(4)->pluck('id')->toArray();
-                $classifications = Classification::inRandomOrder()->limit(3)->pluck('id')->toArray();
+                $classifications = Classification::take(5)->inRandomOrder()->limit(3)->get()->pluck('id')->toArray();
                 $user->cmoAssets()->sync($assets);
                 $user->expectedClassifications()->sync($classifications);
+                PoolCandidate::factory()->count(1)->sequence(fn () => [
+                    'pool_id' => Pool::pluck('id', 'key')->toArray()['digital_careers'],
```

Comment:
Not a big deal but this is a bit sloppy.  Rather than querying the db for every key and selecting on you can query for just the one you need.